core/tests/unit/ldml/meson.build

```diff
@@ -33,7 +33,7 @@ ldml = executable('ldml',
     objects: lib.extract_all_objects())
 
 tests = [
-  '000_null_keyboard',
+  # '000_null_keyboard',
```

Comment:
We should still have a null_keyboard test that just loads and exits without any keystrokes, ideally?

Reply:
The validator will eventually reject it as not containing any sections. I could update it to be valid but null.

Reply:
Yep, updating it to be a valid null keyboard would be good.